### PR TITLE
Middle section works and about myself

### PR DESCRIPTION
**Pull request by lenkon**
Project name: Portfolio: finish mobile version

Project requirements-
Build the middle section of the mobile website by adding the "Works" section which is a list of projects, and the "About myself" section in the mobile portfolio website.

Project requirements link: https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,34 +23,225 @@
...
+          </ul>
+          <nav class="card-button-box">
+            <button class="card-button">See Project</button>
+          </nav>        
+        </div>
+      </section>
+
+      <!-- About Myself -->
+      <section class="about-myself">
+        <h1 id="about-myself-header">About <br />Myself</h1>
```
Comment (joskalenda) [inline index.html]: - [ ] Well done @lenkon, your code is looking clean regarding the requirement, but we have a few things to modify in the project styling, as mentioned in the requirement, please make your best to match as much as possible the design, the `About me` title is not same as the one in the original design, please fix that up.
- [ ] Take a look closer at the technologies section, they have to be aligned on the same line with the button, kindly fix that up.

sbt

File: index.html [hunk 2]
```diff
@@ -21,34 +23,225 @@
...
+      </section>
+
+      <!-- Middle section - Works -->
+      <section class="card-works-wrapper">
+
+        <!-- Card 1 -->
+        <div class="card-works-container card-size1">
+          <img src="./image/Snapshoot-Portfolio.png" alt="Project1">
+          <h1 class="card-title-text">Tonic</h1>
+          <div class="card-list">
```
Comment (joskalenda) [inline index.html]: - [ ] Following the Html best practice, a page should only have an `h1` tag, if I remember, on the top of the application, we have used an `h1` already in the previous milestone. kindly change this one into an `hx` other than h1

File: index.html [hunk 3]
```diff
@@ -21,34 +23,225 @@
...
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="card-tools-list">
+            <li class="card-tools">html</li>
+            <li class="card-tools">css</li>
+            <li class="card-tools">javascript</li>
+          </ul>
+          <nav class="card-button-box">
+            <button class="card-button">See Project</button>
+          </nav>
```
Comment (joskalenda) [inline index.html]: - [ ] - [ OPTIONAL ]  Everything is fine here but I don't think it is the best way to use the nav tag in the middle of the page just as a container, rather you use a div, please make sure you modify it later.

File: index.html [hunk 4]
```diff
@@ -21,34 +23,225 @@
...
+        </div>
+
+        <!-- Card 2 -->
+        <div class="card-works-container card-size2">
+          <img src="./image/Snapshoot-Portfolio-Availibility.png" alt="Project2">
+          <h1 class="card-title-text">Multi-Post </br> Stories</h1>
```
Comment (joskalenda) [inline index.html]: - [ ] Same thing applies for this h1, please change it into an `hx..(h2,3,4......)` instead of h1

File: index.html [hunk 5]
```diff
@@ -21,34 +23,225 @@
...
+          <nav class="card-button-box">
+            <button class="card-button">See Project</button>
+          </nav>        
+        </div>
+
+        <!-- Card 3 -->
+        <div class="card-works-container card-size1">
+          <img src="./image/Snapshoot-Portfolio-Gym-Fit.png" alt="Project3">
+          <h1 class="card-title-text">Tonic</h1>
+          <div class="card-list">
```
Comment (joskalenda) [inline index.html]: - [ ] Same as here

File: index.html (hunk 1 above)
Comment (joskalenda) [inline index.html]: - [ ] Same as here

File: index.html [hunk 6]
```diff
@@ -21,34 +23,225 @@
...
+          <li>            
+            <a href="https://twitter.com/"><img src="./image/Twitter.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="https://medium.com/"><img src="./image/Vector 2.png" alt="" /></a>
+          </li>                    
+        </ul>        
+        <nav class="card-button-box">
+          <button class="card-button">Get my resume</button>
+        </nav>
```
Comment (joskalenda) [inline index.html]: - [ ] - [ OPTIONAL ]  Everything is fine here but I don't think it is the best way to use the nav tag in the middle of the page just as a container, rather you use a div, please make sure you modify it later.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @lenkon , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]...✅ Green linters
- [ ]...✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (lenkon) [reply index.html]: @joskalenda Thanks for the feedback. I have made the required changes to the "About myself" title. Earlier the font-family link was incorrect. So I have corrected it. I have also corrected the technologies section alignment.

File: index.html (hunk 2 above)
Comment (lenkon) [reply index.html]: @joskalenda, I have changed the h1 tag to h2 tag. Thanks.

File: index.html (hunk 3 above)
Comment (lenkon) [reply index.html]: Hi @joskalenda, thanks for the tips. I have replaced all the nav tags in the middle of the page with the div tags.

File: index.html (hunk 4 above)
Comment (lenkon) [reply index.html]: Hi @joskalenda, thanks for the feedback. I have replaced the h1 tag with an h2 tag.

File: index.html (hunk 5 above)
Comment (lenkon) [reply index.html]: @joskalenda, I have replaced the h1 tag with an h2 tag here as well. Thanks.

File: index.html (hunk 1 above)
Comment (lenkon) [reply index.html]: @joskalenda, thanks again. I have replaced the tag with an h2 tag.

File: index.html (hunk 6 above)
Comment (lenkon) [reply index.html]: @joskalenda I have replaced the nav tag with a div tag. Thanks for the feedback and the tips.

**Review by amejid — APPROVED** [review]
Hi @lenkon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good UI design :fire: 
- Correct use of Github flow :heavy_check_mark: 
- Good commit messages :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**